lib/vmtranslator/translator.go

```diff
@@ -0,0 +1,141 @@
+package vmtranslator
+
+import (
+	"log"
+	"strconv"
+	"strings"
+)
+
+type AC_TYPE int
+
+const (
+	AC_OP AC_TYPE = iota
+	AC_RELOP
+	AC_VAR
+	AC_JUMP
+)
+
+type AsmCommand_t struct{}
+
+type AsmCommand interface{}
+
+func TranslateCommand(cmd *Command) (string, error) {
```

Comment:
I agree. No need for error handling at this stage. Or if there is an error then log and panic and die

Reply:
we'll log the error with log.fatal and a helpful error message for us to debug whatever went wrong. no buffer for backend needed